app/components/groups/group-storage.service.js

```diff
@@ -19,21 +19,27 @@
         "panel": panelLoader
       }
 
+      var loadResources = {
+        "groups.list": GroupResource,
+        "trash.list": TrashResource
+      }
+
       function loadData(params){
         var view = sessionStorage.getItem("userView")
-        loaders[view](params)
+        var loadResource = loadResources[$state.current.name]
```

Comment:
it makes code confusing and less readable, less searchable than direct calls of services